include/ofi_util.h

```diff
@@ -546,79 +551,52 @@ int ofi_cq_signal(struct fid_cq *cq_fid);
 const char *ofi_cq_strerror(struct fid_cq *cq, int prov_errno,
 			    const void *err_data, char *buf, size_t len);
 
-int ofi_cq_write_overflow(struct util_cq *cq, void *context, uint64_t flags,
-			  size_t len, void *buf, uint64_t data, uint64_t tag,
-			  fi_addr_t src);
-
-
-static inline void
-ofi_cq_write_entry(struct util_cq *cq, void *context, uint64_t flags,
-		   size_t len, void *buf, uint64_t data, uint64_t tag)
+static inline int
+ofi_cq_write(struct util_cq *cq, void *context, uint64_t flags, size_t len,
+	     void *buf, uint64_t data, uint64_t tag, uint64_t src)
```

Comment:
If you make this change up front, it may simplify the change set.  The util_cq has a read_entry function.  Adding a write_entry equivalent, based on FI_SOURCE, should be straightforward. That's essentially what peer_cq->owner_ops->write() is doing once the peer cq support ends up being added.